packages/selection/src/useSelection.js

```diff
@@ -0,0 +1,270 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import { useEffect, useContext, useReducer } from 'react';
+
+import { LocaleContext } from './components/LocaleProvider';
+import { composeEventHandlers } from './utils/composeEventHandlers';
+import { getControlledValue } from './utils/getControlledValue';
+import { KEY_CODES } from './utils/KEY_CODES';
+import { DIRECTIONS } from './utils/DIRECTIONS';
+import { ACTIONS } from './utils/ACTIONS';
+
+/* eslint-disable indent */
```

Comment:
Is this necessary anymore? We might be able to tweak the `.eslintrc` file to avoid